.idea/vcs.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hey, @lordent 
Maybe this file is unnecessary.

Reply:
@MrXu- you're right, I did it in a hurry